src/api/__tests__/production-api-client.test.ts

```diff
@@ -2,11 +2,12 @@ import { createProductionApiClient } from '../production-api-client';
 import { createServer, Request, Response } from 'miragejs';
 import { ReportingConfigApiResponse } from '../types';
 import { LogPayload } from '../../monitoring/types';
+import { Server } from 'pretender';
 
 describe( '[ProductionApiClient]', () => {
 	const fakeNonce = 'abc123';
 	const fakeNonceHeaderName = 'x-fake-nonce';
-
+	const fakeCacheKey = 'cachedReportingConfigData';
```

Comment:
**Nit (non-blocking)**: tiny nit, but this is actually the real cache key! So I think we can call this just `cacheKey`. It just made me pause for a second and wonder if we were somehow injecting this!